gradio/_simple_templates/simpletextbox.py

```diff
@@ -0,0 +1,100 @@
+"""gr.Textbox() component."""
+
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from gradio.components.base import FormComponent
+from gradio.events import Events
+
+
+class SimpleTextbox(FormComponent):
+    """
+    Creates a textarea for user to enter string input or display string output.
+    Preprocessing: passes textarea value as a {str} into the function.
+    Postprocessing: expects a {str} returned from function and sets textarea value to it.
+    Examples-format: a {str} representing the textbox input.
+    """
+
+    EVENTS = [
+        Events.change,
+        Events.input,
+        Events.select,
+        Events.submit,
+        Events.focus,
+        Events.blur,
+    ]
```

Comment:
Could we slim this down? We could maybe get rid of a few events and just have 2/3 submit/ change related ones?

Reply:
Word. Keeping change, input, and submit!